pki/generate

```diff
@@ -172,7 +172,7 @@ Dir.chdir(output_dir) do
 
   proxy_node_vars = {
     'HUB_ENTITY_ID' => options.hub_entity_id,
-    'HUB_URL' => "#{options.idp_url}/stub-idp-demo/SAML2/SSO",
+    'HUB_URL' => "#{options.idp_url}/SAML2/SSO",
```

Comment:
We might want to change idp_url to hub_url in options.rb going forward.

Reply:
We probably shouldn't really be using the script at all...